apps/base/rucio-daemons/cms-rucio-daemons.yaml

```diff
@@ -93,6 +93,13 @@ conveyorTransferSubmitter:
   additionalEnvs:
     - name: X509_USER_PROXY
       value: "/opt/proxy/x509up"
+  threads: 5
```

Comment:
Here is the thread config for the submitter, this keeps all its config in a single place